### ISSUE TITLE
`Badges` sort options are not in plain English

### ISSUE THREAD

**Issue by ZeroPyrozen** [Osekai/osekai#60]
Sort options in `Badges` menu are still in variables name instead of its value.

![](https://pyrozen.s-ul.eu/GP8uJLZk)

**Comment by minusQuantumNeko** [comment]
Works fine for me, are you sure it's not `Ctrl+F5` issue? Try resetting cache

**Comment by Tanza3D** [comment]
depends on internet speed, and computer speed. slower computers on slower internet will have this happen more, and even then it seems like random chance. i know the issue.

**Comment by EXtremeExploit** [comment]
maybe making `GetStringRawNonAsync` private (putting it inside `GetStringRaw`) and make use of `GetStringRaw` instead everywhere in code so that this doesn't happen anymore :thinking:

**Comment by Tanza3D** [comment]
won't help. it needs to wait for the file to be grabbed from JS

**Comment by EXtremeExploit** [comment]
it does wait until the resource if loaded tho
![image](https://user-images.githubusercontent.com/30161277/204675711-edf5af72-1dc5-4594-9c59-6f2e6821242a.png)

**Comment by Tanza3D** [comment]
yeah, but it's an async function, so it has to be `await`ed, and the function it's being ran in has to be async too. in badges it's not.

**Comment by EXtremeExploit** [comment]
GetStringRaw is async, and not necesarilly the parent has to be async too, it will `loadIfSourceNotLoaded` and not execute the return until its finished loading the source, you only have to use the await on promises which it does use so `GetStringRaw` is doing exactly what its doing i think

**Comment by EXtremeExploit** [comment]
nvm
